web/packages/teleport/src/Integrations/Enroll/AwsOidc/AwsOidc.test.tsx

```diff
@@ -125,7 +125,7 @@ test('generate command', async () => {
 
   // Test create is still called with 404 ping error.
   jest.clearAllMocks();
-  let error = new ApiError('', { status: 404 } as Response);
+  let error = new ApiError('', { status: 404 } as Response, null);
```

Comment:
I think the `proxyVersion` in `ApiError` could be optional, so we wouldn't have to pass `null` everywhere.

Reply:
i made it into an object param